qic2.py

```diff
@@ -85,7 +87,10 @@ def doTagging(imageList, startTag, endTag, summary):
                     )
                     pywikibot.showDiff(oldtext, text)
         else:
-            print("Oops " + image.encode("utf-8") + " doesn't exist...")
+			try:
+				print("Oops " + image.encode("utf-8") + " doesn't exist...")
+			except:
+				print("Error with" + str(image))
```

Comment:
```suggestion
			print("Oops " + image + " doesn't exist...")
```
Didn’t test this, but based on the error message, it looks like the bot is running under Python3, and `image` is a `str` object (used to be `unicode` under Python2). String literals are also `str`s, so you can just skip encoding it. With this proposed change, it’ll stop working under Python2, but no one should use Python2 in 2022 anyways.